src/examples/rest/src/backend/schema/auth/forgotten-password/entity.ts

```diff
@@ -0,0 +1,21 @@
+import {
```

Comment:
Why do we have so many "entities" (data or ORM) for forgotten password?/ Any authentication entity.

There's one here "src/examples/rest/src/backend/schema/auth/forgotten-password/entity.ts"

There's the Authentication entity in the MySQL db.  This Forgotten Password is one of these Authentications.
"src/backend/entities/mysql/authentication.ts"

There's another Authentication entity in:
"src/packages/auth/src/authentication/entities/authentication.ts"

Is one of these not needed?